pkg/client/client.go

```diff
@@ -208,7 +237,7 @@ func getServerURL(execCredential *datamodel.ExecCredential) (string, error) {
 	return server, nil
 }
 
-func getTLSConfig(pemCAs []byte, nextProto string, insecureSkipVerify bool) (*tls.Config, error) {
+func (c *clientHelpersImpl) GetTLSConfig(pemCAs []byte, nextProto string, insecureSkipVerify bool) (*tls.Config, error) {
```

Comment:
could we theoretically just generate some dummy pemCAs and test that way? I've done something similar quite a lot for testing the server implementation

Reply:
i'd rather avoid making superfluous interfaces only for mockability as much as possible